Need to retain FamilyInternetV6 for backwards compatibility with CDE and other
sources written to the original X11/IPv6 draft which was included in Solaris 9.

diff --git a/X.h b/X.h
index 5cf695d..d06886b 100644
--- a/include/X11/X.h
+++ b/include/X11/X.h
@@ -299,6 +299,7 @@ are reserved in the protocol for errors and replies. */
 #define FamilyDECnet		1
 #define FamilyChaos		2
 #define FamilyInternet6		6	/* IPv6 */
+#define FamilyInternetV6	6	/* SUNSOFT - retained for backwards compatibility with original IPv6 draft  */
 
 /* authentication families not tied to a specific protocol */
 #define FamilyServerInterpreted 5
